tests/manage/monitoring/conftest.py

```diff
@@ -232,38 +233,64 @@ def stop_osd():
 
 
 @pytest.fixture
-def measure_corrupt_pg(measurement_dir):
+def measure_corrupt_pg(request, measurement_dir):
     """
     Create Ceph pool and corrupt Placement Group on one of OSDs, measures the
     time when it was corrupted and records alerts that were triggered during
     this event.
 
     Returns:
         dict: Contains information about `start` and `stop` time for
-        corrupting Ceph Placement Group
+            corrupting Ceph Placement Group
     """
-    oc = ocp.OCP(
-        kind=constants.DEPLOYMENT, namespace=config.ENV_DATA.get("cluster_namespace")
-    )
-    osd_deployments = oc.get(selector=constants.OSD_APP_LABEL).get("items")
-    osd_deployment = osd_deployments[0].get("metadata").get("name")
+    osd_deployment = deployment.get_osd_deployments()[0]
+    original_deployment_revision = osd_deployment.revision
     ct_pod = pod.get_ceph_tools_pod()
     pool_name = helpers.create_unique_resource_name("corrupted", "pool")
     ct_pod.exec_ceph_cmd(f"ceph osd pool create {pool_name} 1 1")
+    ct_pod.exec_ceph_cmd(f"ceph osd pool application enable {pool_name} rbd")
+
+    def teardown():
+        """
+        Make sure that corrupted pool is deleted and ceph health is ok
```

Comment:
Thank you for taking care of removing the corrupted pool and making sure that ceph health is ok upon failure